concepts/numbers/about.md

```diff
@@ -55,6 +55,75 @@ There are three types of maximum (and minimum / maximum negative) values for num
 
 Because of how numbers in JavaScript are implemented, **not** every number between `Number.MIN_VALUE` and `Number.MAX_VALUE` can be represented.
 However, _every_ number between `Number.MIN_SAFE_INTEGER - 1` and `Number.MAX_SAFE_INTEGER + 1` **can** be represented.
+## Infinity - Positive and Negative 
```

Comment:
Currently your headlines feel like they have different levels. "Built-in Objects" is an h2 header and so is "Infitity" or "NaN" although one a very wide field and one is a very detailed small-ish aspect.

This point is a bit hard to explain but maybe you understand what I mean. I would recommend you create one h2 `Special Built-In "Numbers"` and but the content for infinity and NaN below.